clients/client-python/gravitino/api/supports_schemas.py

```diff
@@ -0,0 +1,126 @@
+"""
+Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2.
+"""
+from abc import ABC, abstractmethod
+from typing import List, Dict, Optional
+
+from gravitino.api.schema import Schema
+from gravitino.api.schema_change import SchemaChange
+from gravitino.name_identifier import NameIdentifier
+from gravitino.namespace import Namespace
+
+
+class NoSuchSchemaException(Exception):
+    """Exception raised if the schema does not exist."""
+    pass
+
+
+class SupportsSchemas(ABC):
```

Comment:
This interface will be changed soon(plan in 0.6, see https://github.com/datastrato/gravitino/issues/553 and https://github.com/datastrato/gravitino/issues/2479); I'm not suggesting to implement it now in Python.

Reply:
When we create  Fileset catalog or do other operations, We need create metalake and schema before that, Because Fileset catalog dependence metalake and schema.
So I need implement SupportsSchemas, I don't wait 0.6 version.